lib/puma/app/status.rb

```diff
@@ -55,6 +55,11 @@ def call(env)
 
         when /\/stats$/
           rack_response(200, @cli.stats)
+
+        when /\/thread-backtraces$/
+          strings = Puma::Events.strings
+          @cli.log_thread_status(strings)
+          rack_response(200, strings.stdout.string)
```

Comment:
Should this be JSON instead of the raw string?

Reply:
Yes. All current responses are json, so we're going to need to turn this into json object.

Maybe something like:

```
threads: [
  { name: "", backtrace: "" },
  { name: "", backtrace: "" }
]
```